app-misc/mx5000tools/mx5000tools-0.1.2.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+inherit autotools
+
+DESCRIPTION="Tools for controlling the LCD on a Logitech MX5000 keyboard"
+HOMEPAGE="https://web.archive.org/web/20160409073317/http://home.gna.org/mx5000tools/"
+SRC_URI="https://web.archive.org/web/20170225160711/http://download.gna.org/${PN}/${P}.tar.gz"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~ppc64 ~x86"
+IUSE="static"
+
+DEPEND="media-libs/netpbm:="
```

Comment:
…and doesn't it require static-libs netpbm?

Reply:
Seems not the case, as it works just fine. The tools always use the dynamic lib, not the static one.